packages/typespec-vscode/README.md

```diff
@@ -85,3 +85,7 @@ This setting provides the ability to configure where the tsp compiler is located
   "typespec.tsp-server.path": "${workspaceFolder}/my-nested-project/node_modules/@typespec/compiler"
 }
 ```
+
+### Telemetry
+
+The TypeSpec Extension for Visual Studio Code collects usage data and sends it to Microsoft to help improve our products and services. Read our [privacy statement](https://privacy.microsoft.com/privacystatement) to learn more. This extension respects the `telemetry.telemetryLevel` setting which you can find more information in the [FAQ](https://code.visualstudio.com/docs/supporting/faq#_how-to-disable-telemetry-reporting).
```

Comment:
are we able to list explicitly what we collect

Reply:
This is mainly for data privacy statement. it will ultimately present [here](https://marketplace.visualstudio.com/items?itemName=typespec.typespec-vscode#:~:text=The%20TypeSpec%20extension%20for%20Visual%20Studio%20Code%20enhances,IntelliSense%20%26%20Auto-completion%3A%20Code%20faster%20with%20smart%20suggestions). Do we need these details in the overview page, or shall we set up the details in a sub session? Or leave a link to a github page with details?

Reply:
can we merge the PR first. I will draft another PR to update this page https://typespec.io/docs/introduction/editor/vscode/ with more details about data and telemetries we collected in vs code extension.